app/uk/gov/hmrc/eusubsidycompliancefrontend/services/SubsidyJourney.scala

```diff
@@ -91,4 +93,16 @@ object SubsidyJourney {
     Json.format[FormPage[TraderRef]]
 
   implicit val format: Format[SubsidyJourney] = Json.format[SubsidyJourney]
+
+  def fromSubsidy(nonHmrcSubsidy: NonHmrcSubsidy): SubsidyJourney = {
+    val newJourney = SubsidyJourney()
+    newJourney
+      .copy(reportPayment = newJourney.reportPayment.copy(value = Some(true)))
```

Comment:
You should be able to do this with a single `.copy` on `newJourney` and multiple arguments e.g.

```
newJourney
  .copy(
    foo = .....,
    bar = .....
  )
```